tracks/python/exercises/reverse-string/mentoring.md

```diff
@@ -0,0 +1,74 @@
+### Problem and challenges
+
+The problem requires the student to reverse a given string.
+
+### Reasonable solutions
+
+[Extended Slice Notation][1] is the fastest way to reverse a string in Python
+
+```python
+def reverse(text):
+    return text[::-1]
+```
+
+Though it can be tricky for beginners to comprehend, also some developers
+found it not quite readable, so
+
+```python
+def reverse(text):
+    return "".join(reversed(text))
+```
+
+is also a good straightforward solution with a decent performance.
```

Comment:
To visibly show that the sentence has been interrupted by the code example put an ellipsis at the front of this string, ie "... is also a".

Also, I would suggest replacing "good" with "reasonable" and "decent" with "acceptable" ... this is **definitely** the less idiomatic and less performant solution, we should accept it, but not promote it.